src/FitbitPHPOAuth2.php

```diff
@@ -21,7 +15,7 @@
  * @version 1.0.0 ($Id$)
  * @license http://opensource.org/licenses/MIT MIT
  */
-class FitBitPHPOauth2 {
+class FitbitPHPOauth2 {
```

Comment:
This is the public facing `interface` for this and if it has to remain `FitBit` with a capital `B` I think that's fine.